src/main/java/gregtech/common/blocks/wood/BlockRubberDoor.java

```diff
@@ -0,0 +1,43 @@
+package gregtech.common.blocks.wood;
+
+import net.minecraft.block.BlockDoor;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+
+import java.util.function.Supplier;
+
+public class BlockRubberDoor extends BlockWoodenDoor {
```

Comment:
Could this just be merged into BlockWoodenDoor instead of extending it? This all seems like a standard impl which would be beneficial to other doors.

Reply:
The reason I've inherited the class is that the rubber door has slightly less thickness than other doors. As treated wood does not need custom collisionbox, I've separated them using inheritance.